### ISSUE TITLE
I don't even know what this is...

### ISSUE THREAD

**Issue by Starcatcher2009** [GarwelGarwel/KerbalHealth#169]
[EXC 15:39:44.952] NullReferenceException: Object reference not set to an instance of an object
    KerbalHealth.CLS.GetCLSSpace (ProtoCrewMember pcm, Vessel vessel) (at <01bb8e04623f43048f84145489c3e8ec>:0)
    KerbalHealth.Core.GetCrew (ProtoCrewMember pcm, System.Boolean entireVessel) (at <01bb8e04623f43048f84145489c3e8ec>:0)
    KerbalHealth.Logic.Test (ProtoCrewMember pcm) (at <01bb8e04623f43048f84145489c3e8ec>:0)
    KerbalHealth.Logic.Test (ProtoCrewMember pcm) (at <01bb8e04623f43048f84145489c3e8ec>:0)
    (KerbalHealth.HealthCondition hc) (at <01bb8e04623f43048f84145489c3e8ec>:0)
    System.Linq.Enumerable+WhereEnumerableIterator`1[TSource].MoveNext () (at <351e49e2a5bf4fd6beabb458ce2255f3>:0)
    KerbalHealth.KerbalHealthScenario.UpdateKerbals (System.Boolean forced) (at <01bb8e04623f43048f84145489c3e8ec>:0)
    KerbalHealth.KerbalHealthScenario.OnSave (ConfigNode node) (at <01bb8e04623f43048f84145489c3e8ec>:0)
    ScenarioModule.Save (ConfigNode node) (at <4b449f2841f84227adfaad3149c8fdba>:0)
    ProtoScenarioModule..ctor (ScenarioModule module) (at <4b449f2841f84227adfaad3149c8fdba>:0)
    ScenarioRunner.UpdateModules () (at <4b449f2841f84227adfaad3149c8fdba>:0)
    ScenarioRunner.GetUpdatedProtoModules () (at <4b449f2841f84227adfaad3149c8fdba>:0)
    Game.Updated (GameScenes startSceneOverride) (at <4b449f2841f84227adfaad3149c8fdba>:0)
    GamePersistence.SaveGame (System.String saveFileName, System.String saveFolder, SaveMode saveMode, GameScenes startScene) (at <4b449f2841f84227adfaad3149c8fdba>:0)
    GamePersistence.SaveGame (System.String saveFileName, System.String saveF
KerbalHealth.MTBEModifier.Calculate (System.Double baseValue, ProtoCrewMember pcm) (at <01bb8e04623f43048f84145489c3e8ec>:0)
    KerbalHealth.MTBEModifier.Calculate (System.Collections.Generic.List`1[T] modifiers, System.Double baseValue, ProtoCrewMember pcm) (at <01bb8e04623f43048f84145489c3e8ec>:0)
    KerbalHealth.HealthCondition.GetMTBE (ProtoCrewMember pcm) (at <01bb8e04623f43048f84145489c3e8ec>:0)
    KerbalHealth.KerbalHealthScenario+<>cDisplayClass64_2.<UpdateKerbals>b1

Every UI button just stops working.

**Comment by Phonetic-Decay** [comment]
I encountered the exact same issue where this conflict series was running on a loop in orange. I couldnt tell what mod it was due to some kind of generic conflict [cannot reference] which led me to uninstall mods piecemeal. The next time it happened the second conflict was gone and this was in it place, and able to see the syntax Kerbal Health.

I believe this is conflict with other mods but i havent a clue. If Dev needs, i will down load the extended logger from CKAN and see if i can a more comprehensive log. If no one gets back for another few months... i imagine its the For Science! on KSP2 and ill ask ChatPT ;p
Kuddos on such an awesome mod, and closing so many bugs.
